### DISCUSSION TITLE
Browser extension example

### DISCUSSION THREAD

**Discussion by killix** [sveltejs/kit#4798] [General]
I am looking for a web extension starter template to create a browser extension. I struggling with the CSP option with svelte-kit and cannot find a way to make it work in a chrome extension environment.
Someone can point me on the good direction?

Thanks.

**Comment by geoffrich** [comment]
I haven't used it myself, but @antony made an [SvelteKit adapter](https://github.com/antony/sveltekit-adapter-browser-extension) that turns your app into a browser extension. Looks like it [includes CSP support](https://github.com/antony/sveltekit-adapter-browser-extension/blob/a377dd83b4ecd599442725b680bed490b9163a97/adapter-browser-extension.mjs#L14-L22).

**Reply by killix** [reply]
thanks, I saw it. But I have some trouble making the webextension-polyfill work with this adapter.
